CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
-# New feature
-- Addition of `class_id` to `quickbooks__general_ledger` and `quickbooks_general_ledger_by_period`, add in class values for all intermediate models necessary to pass into final models.
+# New Features (In Progress)
+
+## 🚨 Breaking Changes 🚨:
+- Added `transaction_source` to surrogate key function to fix `unique_id` uniqueness issues.  Customer will need to do either a full refresh or add in the variable to their modeling to produce up-to-date `unique_id` values.
```

Comment:
Sorry for the back and forth for this. This is just a pretty big change that I think should be highlighted to the best of our ability. 

If we release this as part of the migration branch, I think the messaging should be very similar to what we currently have in the changelog re: generating surrogate keys. 

> dbt_utils.surrogate_key has also been updated to dbt_utils.generate_surrogate_key. Since the method for creating surrogate keys differ, we suggest all users do a full-refresh for the most accurate data. For more information, please refer to dbt-utils [release notes](https://github.com/dbt-labs/dbt-utils/releases) for this update.

Maybe this specific update could be a sub-bullet of the surrogate key note above from the migration branch. I mostly want to make sure that customers know that there is a variable they may need to toggle.

Reply:
Hey! Sorry for the confusion.

This branch will not be merged into migration, it'll be merged into `feature/quickbooks-updates-project`, which will then get merged onto `main`. 

So either migration will get merged first, or `feature/quickbooks-updates-project`. I think if the latter occurs, it makes sense to do the following above. But right now I think the overall plan is to merge this after migration, which was the reason Joe created this separate branch to work off of.

Sorry if this is a bit roundabout. Happy to discuss more at standup.